js/blocks/components/image.js

```diff
@@ -0,0 +1,151 @@
+const { BaseControl, Button, Placeholder, DropZone, FormFileUpload, Spinner } = wp.components;
+const { withState } = wp.compose;
+const { withSelect } = wp.data;
+const { mediaUpload, MediaUpload, MediaUploadCheck } = wp.editor;
+const { Fragment } = wp.element;
+const { __, sprintf } = wp.i18n;
+
+const ALLOWED_TYPES = [ 'image' ];
+const DEFAULT_IMG_ID = 0;
+
+const Image = withSelect( ( select, ownProps ) => {
```

Comment:
This file is mainly copied from [js/blocks/controls/image.js](https://github.com/getblocklab/block-lab/pull/426/files#diff-7bbb33042f7b94c527dc6a5bdb42c258L1). 

This is because of an ESLint error:

>14:8  error  Variables should not be assigned until just prior its first reference. An early return statement may leave this variable unused  @wordpress/no-unused-vars-before-return

This component was stored in the variable `ImageControl` at line 14 of that file, and not used until [line 155](https://github.com/getblocklab/block-lab/pull/426/files#diff-7bbb33042f7b94c527dc6a5bdb42c258L155).